core/src/main/java/io/confluent/kafka/schemaregistry/rest/resources/ModeResource.java

```diff
@@ -186,6 +189,7 @@ public ModeUpdateRequest updateTopLevelMode(
   }
 
   @GET
+  @DocumentedName("getMode")
```

Comment:
will `getGlobalMode` be a better name here?

Reply:
Addressed